benchmark/sirun/appsec-iast/insecure-bank.js

```diff
@@ -1,4 +1,5 @@
 const http = require('http')
+// eslint-disable-next-line import/no-absolute-path
```

Comment:
If we allow this, why not add it as a global rule?

Reply:
It's some junk for a test that we otherwise wouldn't want i real tracer code